fetchconfig/parseArgs.cpp

```diff
@@ -10,26 +10,19 @@
 
 int parseArgs(int argc, char* argv[], struct option* longOptions, std::function<int (struct option*, int, int)> optionsCaller)
 {
-	int optIndex = 0;
-	int optChar;
-    
+    int optIndex = 0;
+    int optChar;
+
     auto getOpts = [&] () -> int {
```

Comment:
Seems pointless since it is only used once. Remove it and spare 3 lines of code :)